File: backend/btcgram/src/main/java/com/example/btcgram/model/Moment.java
package com.example.btcgram.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Moment {
    private String city;
    private String country;
    private String timezone;
    private double temperature;
    private String condition;
    private String localTime;
    private double btcPriceUsd;
}

File: frontend/src/types/moment.ts
// Mirrors backend/btcgram/src/main/java/com/example/btcgram/model/Moment.java
export type Moment = {
  city: string;
  country: string;
  timezone: string;
  temperature: number;
  condition: string;
  localTime: string;
  btcPriceUsd: number;
};
